src/components/predictive-layer/predictive-layer.vue

```diff
@@ -37,7 +37,7 @@
       >
         <div v-if="showHistoryQueries" class="x-list x-list--gap-02">
           <div v-if="!$x.query.searchBox" class="x-list x-list--horizontal x-list--align-center">
-            <h1 class="x-small x-text--bold x-text--secondary x-list__item--expand">
+            <h1 class="x-small x-text--bold x-list__item--expand">
```

Comment:
Should we use `x-font-color--neutral-35` instead to have the same color of the design?
```suggestion
            <h1 class="x-small x-text--bold x-font-color--neutral-35 x-list__item--expand">
```

Reply:
Oh, okay! Thanks!